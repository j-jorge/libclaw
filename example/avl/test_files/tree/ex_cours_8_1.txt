6 2 8 3 7 4 1 0