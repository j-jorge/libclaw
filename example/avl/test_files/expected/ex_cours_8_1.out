Tree is empty
File loading...
size : 8
Min = 0
Max = 8
Tree's not empty
Found : 0 1 2 3 4 6 7 8
Scan with an iterator : 0 1 2 3 4 6 7 8
Scan bis, iterator affectation :0: 0 1 2 3 4 6 7 8
1: 1 2 3 4 6 7 8
2: 2 3 4 6 7 8
3: 3 4 6 7 8
4: 4 6 7 8
6: 6 7 8
7: 7 8
8: 8
Deletions
Copy and scan
source size : 0
copy size : 0
Tree's copy is empty

Reverse scan

Assignment
Clear
Tree's is empty
